yt/frontends/owls/tests/test_outputs.py

```diff
@@ -26,6 +27,8 @@
 @requires_ds(os33, big_data=True)
 def test_snapshot_033():
     ds = data_dir_load(os33)
+    psc = ParticleSelectionComparison(ds)
```

Comment:
So @jzuhone doesn't need to do the owls, flash, eagle, or arepo tests anymore?

Reply:
right!